karabo_data/tests/mockdata/basler_camera.py

```diff
@@ -0,0 +1,35 @@
+"""Script that creates a mock-run for the basler camera"""
+from os.path import join as op_join
+
+from .base import DeviceBase
+
+class BaslerCamera(DeviceBase):
+    """
+    Basler Camera device
+    Based on example /gpfs/exfel/exp/SPB/201930/p900061/raw/r0055/RAW-R0055-DA01-S00000.h5
+    """
+
+    output_channels = ('daqQutput/data',)
+
+    instrument_keys = [
+        ('image/bitsPerPixel', 'i4', ()),
+        ('image/dimTypes', 'i4', (2,)),
+        ('image/dims', 'u8', (2,)),
+        ('image/encoding', 'i4', ()),
+        ('image/pixels', 'u2', (1029, 1228)), # Odd shape
```

Comment:
did you see that shape in a file? reason is probable due to ROI settings. Either find the native resolution, or set the ROI information accordingly.

Reply:
I guess it's something with the binning. Anyhoo since there are apparently different types of cams out there I added the pixel size a a keyword.